src/main/java/org/medicmobile/webapp/mobile/OpenSettingsDialogFragment.java

```diff
@@ -32,13 +33,10 @@ public boolean onTouch(View view, MotionEvent event) {
 		}
 	};
 
-	public OpenSettingsDialogFragment(View view) {
-		this.view = view;
-	}
-
 	@Override
 	public void onCreate(@Nullable Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
+		View view = getActivity().findViewById(R.id.wbvMain);
 		view.setOnTouchListener(onTouchListener);
```

Comment:
This solution looks so much cleaner! Have you tested that the touch listener still fires and works as expected?

Reply:
Yes, tested this out on a physical device.

Reply:
I also gave it a shot on an emulated device and it still works as expected :+1: